libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/Orphans.hs

```diff
@@ -35,7 +37,7 @@ deriving instance Generic GovEnv
 
 deriving instance Generic EnactState
 
-deriving instance Generic DepositPurpose
+deriving instance Generic Lib.DepositPurpose
```

Comment:
I don't think adding `Lib` is necessary because all of the types here are from `Lib` anyways. I think it'd be better to hide the import of `DepositPurpose` instead if that's causing issues.